docs/developer-guide/coal-setup.md

```diff
@@ -166,12 +166,12 @@ configuration file.
 |Default gateway IP address:|↵|Use the default.|
 |Primary DNS Server|↵|Use the default.|
 |Secondary DNS Server|↵|Use the default.|
-|Head node domain name|example.com||
+|Head node domain name|example.com|Make sure you choose a domain name that is not pingable, or resolved to an unrecognized IP address.|
 |DNS Search Domain|example.com||
 |NTP Server IP Address|↵|Use the default.|
-|"root" password|rootpass||
+|"root" password|rootpass|Enter aphanumeric characters. Do not use space or special characters.|
 |Confirm "root" password|||
-|"admin" password|adminpass1||
+|"admin" password|adminpass1|Enter aphanumeric characters. Do not use space or special characters.|
```

Comment:
Password parsing was fixed in OS-7361 (joyent/sdc-headnode@ b5c0e5a5f6e0ed31c6db2573de9b720d851af603), so these comments on passwords are unnecessary (and dangerous).